### ISSUE TITLE
tutorials/Model_Quickload.ipynb is compatible with models?

### ISSUE THREAD

**Issue by hojongs** [facebookarchive/caffe2#1525]
I'm following Loading Pre-trained model in tutorials.
I have downloaded squeezenet

`python -m caffe2.python.models.download -i squeezenet`

and I have run Loading_Pretrained_Models.ipynb and Model_Quickload.ipynb.
But I have got error!

`RuntimeError: [enforce fail at operator.cc:53] blob != nullptr. op Conv: Encountered a non-existing input blob: data`

I think squeezenet needs data blob before CreateNet(), but tutorial code doesn't do.
So, I modified Code

before call "workspace.CreateNet(predict_net)", 
insert code "workspace.FeedBlob('data', img)"
And other error occured

`RuntimeError: [enforce fail at conv_op_impl.h:53] filter.dim32(i + 2) == kernel_[i].  Error from operator: 
input: "data" input: "conv1_w" input: "conv1_b" output: "conv1" type: "Conv" arg { name: "stride" i: 2 } arg { name: "pad" i: 0 } arg { name: "kernel" i: 3 } device_option { } engine: ""`

And I try it, finally I have got correct output

`results = p.run([]) # alternative p.run([img])`

`Raw top 3 results: [array([985.0, 0.975209653377533], dtype=object), array([309.0, 0.01531461626291275], dtype=object), array([946.0, 0.008236425928771496], dtype=object)]
The image contains a daisy with a 97.5209653378 percent probability.`

what's wrong in tutorial code and model?
I wonder what is the role of parameter of p.run([])
I think it is input data but input data is already feeded by FeedBlob()

In conclusion,
1. Why tutorial code makes error without modifying the code? It is surely compatible with models?
if not, did I anything wrong?
2. What should I give the parameter to p.run()?

Forgive me for my lack of English ;)

**Comment by hojongs** [comment]
I fixed it, I did git clone the model from https://github.com/caffe2/models,
instead use caffe2.python.models.download module
It works well

**Comment by CriCL** [comment]
Hi @ssaemo , could you please share your code that worked?

I'm facing the same issue.

Regards
